axes/handlers/database.py

```diff
@@ -119,58 +119,55 @@ def user_login_failed(
         # 2. database query: Calculate the current maximum failure number from the existing attempts
         failures_since_start = 1 + self.get_failures(request, credentials)
```

Comment:
Looking the code closely this value calculation is not really useful: it is only used in the log message later, and that information is available on the `attempt.failures_since_start` field, I think we should use that instead of this pseudovalue and just mark this whole function with `@transaction.atomic`?

Reply:
The query get_failures is responsible to find similar attempts (e.g. same ip different user), so needs to return to the condition. Sorry my bad, I did not setup the tests for me locally in time.

Reply:
Ah, true that. So, let's keep it in place for the time being.